docs/administrator-documentation/moderne-dx/how-to-guides/dx-org-service.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_label: Creating an organizations service
+description: How to create and configure an organizations service that pairs with Moderne DX.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Creating an organizations service
+
+You should create a dedicated organizations service if you want to:
+
+* Limit access to the organizations you've [previously defined](./configure-dx-organizations.md) so that some users only have access to some repositories OR
+* Customize commit messages by repository (e.g., adding a JIRA ticket to your commit messages based on the repository)
```

Comment:
As above, DX/CLI doesn't have the ability to customize commit messages, so therefore I would do the following:

```suggestion
You should create a dedicated organizations service if you want to limit access to the organizations you've [previously defined](./configure-dx-organizations.md) so that some users only have access to some repositories.
```